### DISCUSSION TITLE
Encoder orientation

### DISCUSSION THREAD

**Discussion by Estraxtos** [tttapa/Control-Surface#865] [Q&A]
Hi, it´s me again, in my control surface code, i have two encoders but the orientation it´s upside down, i mean 0 right and left 127 the midi value, there´s no way to change that? ( correct reference right 127, left 0)

**Comment by Trickster-git** [comment]
[CCRotaryEncoder](https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/d62/classCCRotaryEncoder.html) enc {

  {2, 3},       // pins replase 2,3 to {3, 2}


  [MCU::V_POT_1](https://tttapa.github.io/Control-Surface-doc/Doxygen/d0/d7e/group__MCU__CC.html#ga9400848eec9735623e1502aef5bb7cb3), // MIDI address (CC number + optional channel)
  1,            // optional multiplier if the control isn't fast enough
};

**Reply by tttapa** [reply]
Indeed, you can just swap the pins, either in software or in hardware.
